### PR TITLE
fix of issue no #71

### LINKED ISSUE #71: Not Found Page should be added
If you navigate to any non existing path;; for example : https://kucc.ku.edu.np/abc
There should be a page displaying PageNotFound or something like that
But instead nothing is rendered
KUCC_BUGG

### PR DESCRIPTION
**Pull request by razzat008**
- Added routes for non-existing pages/urls 
- Added png for the error code-404

fixes #71

### CODE REVIEW

**Comment by abhiyandhakal** [comment]
@razzat008 thank you for your PR. Please include the issue number in the body of the pull request (also). For now, I've edited it, if you don't mind.

File: src/pages/PageNotFound.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <div className="flex flex-col items-center justify-center h-screen">
+      <img
+        src="/assets/images/404-image.png" // Replace with the path to your 404 image
+        alt="Could not load image"
+        className="w-64 h-auto mb-4"
+      />
+      <div className="text-center">
+        <h1 className="text-4xl font-bold text-gray-900 mb-2">404 Page Not Found</h1>
+        <p className="text-lg text-gray-600">The page you're looking for doesn't exist.</p>
```
Comment (abhiyandhakal) [inline src/pages/PageNotFound.jsx:13]: Also add the link to the homepage

**Review by abhiyandhakal — CHANGES_REQUESTED** [review]
Please make the necessary changes

File: src/pages/PageNotFound.jsx (hunk 1 above)
Comment (razzat008) [reply src/pages/PageNotFound.jsx:13]: I will make the changes
Thanks

**Review by abhiyandhakal — APPROVED** [review]

**Pull request merged**